### ISSUE TITLE
Clarification on different 'msm' binary versions

### ISSUE THREAD

**Issue by mharms** [ecr05/MSM_HOCR#5]
What is the difference between the MSM version available on GitHub (called "version 3.00" in the README, but tagged as "Initial Release 1.0" in GitHub), the version available at this website https://www.doc.ic.ac.uk/~ecr05/MSM_HOCR_v2/ (labelled as "Version 2.00" from Feb 2017), and the `msm` binary that comes with FSL 6.0+ itself?

**Comment by ecr05** [comment]
There are minor differences between https://www.doc.ic.ac.uk/~ecr05/MSM_HOCR_v2/ (labelled as "Version 2.00" from Feb 2017) and the GitHub version 3 - mostly tidying of code. The MSM binary that comes with FSL 6 is, I believe, forked from version2, but it is only compiled with the FastPD libraries. This means it will not run the advanced regularisation options. It is possible to recompile the FSL released source code, with the ELC library to get it to run HOCR. The HCP pipelines use this Github version.

**Comment by mharms** [comment]
Are the "advanced regularisation options" detailed on the FSL Wiki page for MSM?  The README on GitHub page is a bit hard to follow.

**Comment by mharms** [comment]
Also, as a suggestion, it would help avoid confusion regarding versioning in the community if the tag on the released GitHub versions was set to "3.0" rather than "1.0".  @tbbrown

**Comment by ecr05** [comment]
No,  the advanced regularisation options are not on the FSL wiki. I should put together something on my blog. In the meantime there are sample config files which should be quite generic and i'm very happy to support over email. I agree the versioning is very confusing. I'll look into it.
